org.elysium.ui/src/org/elysium/ui/compiler/preferences/CompilerPreferenceInitializer.java

```diff
@@ -30,4 +35,42 @@ public void initializeDefaultPreferences() {
 		preferenceStore.setDefault(CompilerPreferenceConstants.PARALLEL_COMPILES.name(), 5);
 	}
 
+
+	private String getDefaultLilypondPath() {
+		Object startParameterPath = System.getProperties().get("lilypond.path");//$NON-NLS-1$
+		if (startParameterPath != null) {
+			return startParameterPath.toString();
+		}
+
+		String userHome = System.getenv().get("HOME");//$NON-NLS-1$
```

Comment:
I'd rather take only the lilypond.path system property into account and generate the launch config in the Oomph setup with this system property set (like the test launch config).